fixtures/js-stubs/checkinProcessingError.ts

```diff
@@ -0,0 +1,30 @@
+import {CheckinProcessingError} from 'sentry/views/monitors/types';
```

Comment:
I just moved thse to `tests/js/fixtures`. sorry lol

Reply:
ooh yes yes lemme rebase